packages/app/src/components/RevisionComparer/RevisionComparer.tsx

```diff
@@ -23,20 +23,21 @@ const DropdownItemContents = ({ title, contents }) => (
 type RevisionComparerProps = {
   sourceRevision: IRevisionHasPageId
   targetRevision: IRevisionHasPageId
-  currentPageId?: string
-  currentPagePath: string
+  currentPageId: Nullable<string> | undefined
+  currentPagePath: Nullable<string> | undefined
```

Comment:
I think currentPagePath is not be a null able.
Is it possible null?

Reply:
I call the `currentPagePath` from  `useCurrentPagePath()`. The return value allow `undefined`
I will adjust soon